src/components/CustomDateInput/DateInput.tsx

```diff
@@ -0,0 +1,34 @@
+import { Dispatch } from "react"
+import { FilterAction } from "types/CourtCaseFilter"
+
+interface Props {
+  dateType: string
```

Comment:
Since `dateType` can be only `from` or `to`, we could use an `enum` here to be more specific.

Reply:
@C-gyorfi  @monotypical